local.bib

```diff
@@ -69,10 +60,20 @@ @ARTICLE{2020arXiv201005926L
 	volume = {5},
 	number = {6},
 	pages = {143},
-	author = {Megan E. Schwamb and Mario Juri{\'{c}} and Bryce T. Bolin and Luke Dones and Sarah Greenstreet and 
-	Henry H. Hsieh and Laura Inno and R. Lynne Jones and Michael S. P. Kelley and Matthew M. Knight and 
+	author = {Megan E. Schwamb and Mario Juri{\'{c}} and Bryce T. Bolin and Luke Dones and Sarah Greenstreet and
+	Henry H. Hsieh and Laura Inno and R. Lynne Jones and Michael S. P. Kelley and Matthew M. Knight and
 	William T. Reach and Tom Seccull and Colin Snodgrass and David E. Trilling},
 	title = {Year 1 of the Legacy Survey of Space and Time ({LSST}): Recommendations for Template Production to
 	 Enable Solar System Small Body Transient and Time Domain Science},
 	abstract = {}
 }
+
+@misc{RubinDataProductsAbridged, 
```

Comment:
This does not show up correctly in Section A References (I reviewed https://rtn-011.lsst.io/v/PREOPS-1335/index.html). The title is all small-caps, the DOI doesn't appear, and the reference is not hot-linked to Zenodo. 

![Screen Shot 2022-10-20 at 6 21 56 PM](https://user-images.githubusercontent.com/13826801/197089331-fa23d0ef-2600-475c-9559-f9bb9e176bd1.png)

Reply:
And yet, that is what the BibTex generated by Zenodo itself leads to... I guess the LSST bibliography style for "misc" items is the problem here. I'll see if I can improve this so that the reference appears as desired.

Reply:
OK, hacked the BibTex until reference now looks like this:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/710903/197140129-081fdebf-5af4-423d-8c43-ceca32a6d2ca.png">
Not perfect but a lot better! Thanks for catching this, Melissa.